spec/features/choose_lead_image_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+RSpec.feature "Choose a lead image" do
+  scenario "User chooses an existing image as lead image" do
+    given_there_is_a_document_with_existing_images
+    when_i_visit_the_summary_page
+    when_i_visit_the_lead_images_page
+    then_i_should_be_able_to_see_a_list_of_existing_images
+    when_i_remove_the_lead_image
```

Comment:
This looks suspiciously like a separate scenario - can we make it one instead?

Reply:
Fair point 👌